locator/tablets.hh

```diff
@@ -55,15 +55,33 @@ struct tablet_info {
     bool operator==(const tablet_info&) const = default;
 };
 
+enum class tablet_transition_stage {
+    allow_write_both_read_old,
+    write_both_read_old,
+    write_both_read_new,
+    streaming,
```

Comment:
Shouldn't write_both_read_new be after streaming?

Reply:
Yes. Not that it matters for correctness, since transition order is defined elsewhere.

Reply:
Still, it helps reduce confusion.